examples/agent-chatbot/package.json

```diff
@@ -12,8 +12,8 @@
   },
   "dependencies": {
     "@ai-sdk/openai": "^0.0.53",
-    "@mastra/core": "^0.1.12",
-    "@mastra/firecrawl": "^1.0.1",
+    "@mastra/core": "workspace:^",
```

Comment:
shouldn't this be pointing to the published version?

Reply:
yeah it should, we do need to publish though, i'll revert for now